monitoring/osd-managing-metrics.adoc

```diff
@@ -0,0 +1,38 @@
+[id="managing-metrics"]
```

Comment:
There is no content index at the beginning of the page, for detail refer 
https://docs.openshift.com/container-platform/4.7/monitoring/managing-metrics.html

Reply:
This was fixed by a different PR and will render correctly when I commit these other changes. Fixed.